### PR TITLE
Admin User Crud and User List

### PR DESCRIPTION
**Pull request by vanirankar**
Admin User CRUD is implemented and Client User List Created

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (kpvarma) [inline yarn.lock]: Why are we using yarn?

File: app/controllers/admin/client_users_controller.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ class ClientUsersController < Admin::BaseController
     def index
 
       @page_title = "Client Users | Admin"
-
+      @client_users = ClientUser.all
```
Comment (kpvarma) [inline app/controllers/admin/client_users_controller.rb]: Please implement pagination
you can use pagination helper methods I have added to helpers/
just try ClientUser.per(@current_page).page(@per_page)
current_page and per_page will be autopopulated as @ variables if those methods are called in before action

File: db/migrate/20191218062846_addlast_name_toadmin_users.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddlastNameToadminUsers < ActiveRecord::Migration[5.2]
```
[the remark this answers is not in the document]
Comment (vanirankar) [reply db/migrate/20191218062846_addlast_name_toadmin_users.rb]: class AddAdditionalFieldsToUsers < ActiveRecord::Migration[5.2]
  def change
    add_column :admin_users, :first_name, :string
    add_column :admin_users, :last_name, :string
    add_column :admin_users, :phone, :bigint
    add_column :admin_users, :organisation, :string

    add_column :client_users, :first_name, :string
    add_column :client_users, :last_name, :string
    add_column :client_users, :phone, :bigint
    add_column :client_users, :organisation, :string
  end
end

File: lib/templates/erb/scaffold/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
```
[the remark this answers is not in the document]
Comment (vanirankar) [reply lib/templates/erb/scaffold/_form.html.erb]: removed

File: app/controllers/admin/client_users_controller.rb (hunk 2 above)
Comment (vanirankar) [reply app/controllers/admin/client_users_controller.rb]: implemented.

File: yarn.lock (hunk 1 above)
Comment (vanirankar) [reply yarn.lock]: where it is? I was trying to add simple_form.

**Pull request merged**